driver-rabbitmq/src/main/java/io/openmessaging/benchmark/driver/rabbitmq/RabbitMqBenchmarkConsumer.java

```diff
@@ -38,14 +38,13 @@ public RabbitMqBenchmarkConsumer(Channel channel, String queueName, ConsumerCall
 
         this.channel = channel;
         this.callback = callback;
-        channel.basicConsume(queueName, this);
+        channel.basicConsume(queueName, true,this);
```

Comment:
Need a space here, and is this always open automatically ack setting?

Reply:
To measure consumer rate, it is equal to basicAck in handleDelivery below